### ISSUE TITLE
Create personal blog

### ISSUE THREAD

**Issue by Ceachi** [Ceachi/dl-notebooks#2]
Create a personal blog.

Here are some that you can inspire, also please check out their github repos also:

https://haochen23.github.io/2020/06/fine-tune-mask-rcnn-pytorch.html
